From d575089ef8501e46c9ebfa0fa53502f1571c6ef5 Mon Sep 17 00:00:00 2001
From: xfnw <xfnw@riseup.net>
Date: Wed, 11 Dec 2024 11:38:07 -0500
Subject: [PATCH] Revert "update autoconf version requirements to 2.72"

no idea why the minimum autoconf version got randomly bumped, it appears
to still work fine

This reverts commit 866f3bb1eab8de00c21bec225be4670f602d3ded.
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8535969..3a882f0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,7 +1,7 @@
 #                                               -*- Autoconf -*-
 # Process this file with autoconf to produce a configure script.
 
-AC_PREREQ([2.72])
+AC_PREREQ([2.65])
 AC_INIT([gumbo], [0.12.3], [txgk@bk.ru])
 AC_SUBST(gumbo_version_info, [2:2:0])
 AC_CONFIG_MACRO_DIR([m4])
@@ -10,8 +10,8 @@ AC_CONFIG_SRCDIR([src/parser.c])
 AC_CONFIG_FILES([Makefile gumbo.pc])
 
 # Checks for programs.
-AC_PROG_CC
 AC_PROG_CXX
+AC_PROG_CC_C99
 
 # Checks for libraries.
 
-- 
2.47.1

